src/charm.py

```diff
@@ -58,7 +59,9 @@ def __init__(self, *args):
 
         self.metrics_endpoint = MetricsEndpointProvider(
             self,
-            jobs=[{"static_configs": [{"targets": [f"*:{JMX_EXPORTER_PORT}"]}]}],
+            jobs=[
+                {"static_configs": [{"targets": [f"*:{JMX_EXPORTER_PORT}", f"*:{JMX_CC_PORT}"]}]}
+            ],
```

Comment:
Maybe initialize these libs at the bottom, and on L70 do:
```python
self.broker = BrokerOperator(self)
self.balancer = BalancerOperator(self)

self.loki_push = LogProxyConsumer(
            self,
            log_files=[
                f"{self.broker.workload.paths.logs_path}/server.log",
                f"{self.balancer.workload.paths.logs_path}/cruisecontrol.log"
            ],
            alert_rules_path=LOGS_RULES_DIR,
            relation_name="logging",
            container_name="kafka",
)
```

Reply:
Done in e24901fd0d42f2a63d4149f759494e426390ff5e. Thank you, I was missing a log dir